src/utils/gen-serial-j2.py

```diff
@@ -16,6 +16,11 @@
                "depends": {"UART"},
 #               "inverter": True,   # TODO: old code compatibility only, disabled
                },
+    "PIOUART": {"ids": list(range(0, 9 + 1)),
```

Comment:
Maybe place PIOUART after SOFTSERIAL - nothing will change, but it will follow enum (and resource) order